index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav id="nav">
+            <a href="#" class="logo">
+                <p>Bonjour</p>
+            </a>
+            <div><img src="./images/menu.svg" alt="icon_menu" id="menu"></div>
+        </nav>
+    </header>
+    <main>
+        <section class="hero">
+            <div class="hero-content">
+                <div class="hero-title ">
+                    <h1>Hello there <br> I'm Zuhaib <br> Glad to see you!</h1>
+                </div>
+                <div class="hero-para ">
+                    <p>
+                        I am a Software Developer! I will help you in materializing your dream product, feature or
+                        website.
+                        have a look at my work and experience and if you touched by the work i have done, then contact
+                        me
+                        by
+                        clicking the icons below.
+                    </p>
+                </div>
+                <div>
+                    <p class="hero-social">
+                        let's connect
+                    </p>
+                    <div class="hero-icons">
+                        <a target="_blank" href="https://twitter.com/Zuhaib042" rel="noopener noreferrer"><img
+                                src="./images/twitter.svg" alt="twitter-logo"></a>
+                        <a target="_blank" href="https://www.linkedin.com/in/zuhaib-amjad-488914234/"
+                            rel="noopener noreferrer"><img src="./images/linkedin.svg" alt="linkedIn-logo"></a>
+                        <a href="#" rel="noopener noreferrer"><img src="./images/medium.svg" alt="medium-logo"></a>
+                        <a target="_blank" rel="noopener noreferrer" href="https://github.com/Zuhaib042"><img
+                                src="./images/github.svg" alt="github-logo"></a>
```

Comment:
- [ ] Good job wrapping each icon in an `a` tag. Please also wrap all icons in a `ul` tag and each of the icons should be in an `li` tag 🙏

Reply:
I have fixed the changes you suggested